ImageClassification/ViewControllers/ViewController.swift

```diff
@@ -148,7 +150,7 @@ class ViewController: UIViewController {
         resumeButtonSetup()
         liveButtonSetup()
         controlViewSetup()
-
+        
```

Comment:
```suggestion
```

Reply:
Just killing accidental whitespace

Reply:
Only suggesting because I think thats what causes the merge conflicts in the future not the `\n` but I could be wrong